examples/example1/benchmark1/test_fv.py

```diff
@@ -57,29 +54,32 @@ def add_data(gb, domain, kf, mesh_value):
                     np.absolute(g.face_centers[1, bound_faces[left]] - 0.5) < mesh_value
                 )
                 bc_val[bound_faces[left]] = (
-                    -g.face_areas[bound_faces[left]] + left_mid * .5 * a
+                    -g.face_areas[bound_faces[left]] + left_mid * 0.5 * a
                 )
             else:
                 bc_val[bound_faces[left]] = -g.face_areas[bound_faces[left]] * a
 
             bc_val[bound_faces[right]] = np.ones(np.sum(right))
 
-            param.set_bc("flow", pp.BoundaryCondition(g, bound_faces, labels))
-            param.set_bc_val("flow", bc_val)
+            bound = pp.BoundaryCondition(g, bound_faces, labels)
+            specified_parameters.update({"bc": bound, "bc_values": bc_val})
         else:
-            param.set_bc("flow", pp.BoundaryCondition(g, np.empty(0), np.empty(0)))
+            bound = pp.BoundaryCondition(g, np.empty(0), np.empty(0))
 
-        d["param"] = param
+        pp.params.data.initialize_data(d, g, "flow", specified_parameters)
```

Comment:
please use __init__ to make this shorter

Reply:
That has been done already, I just didn't fix it in the tests where I'd already written it out. So just use pp.initialize_data(...).